### PR TITLE
refactor:change the default font from sans-serif to montserrat

### PR DESCRIPTION
**Pull request by CynthiaUmuhire**
@danielerat @gitarecy @Moussa-Kalam 
# Making the default font montserrat
### Checklist before requesting a review
- [x] I made  montserrat the default font that is appplied when you don't specify any  font

### CODE REVIEW

File: src/sections/founders.html [hunk 1]
```diff
@@ -1,22 +1,19 @@
...
-      class="bg-primary bg-gradient-to-r from-transparent from-45% to-secondary/60 text-white px-5 md:px-10 lg:px-20 pt-32"
-    >
+      class="bg-primary bg-gradient-to-r from-transparent from-45% to-secondary/60 text-white px-5 md:px-10 lg:px-20 pt-32">
       <div class="px-mxScreen">
         <div class="max-w-2xl flex flex-col gap-y-4 pb-6">
           <h2 class="text-xl font-bold font-raleway">
             Die <span class="text-secondary">Founder</span> und ihre Vision von
             Ojemba
           </h2>
-          <p class="text-xs font-montserrat">
```
Comment (Moussa-Kalam) [inline src/sections/founders.html:14]: Great job on removing `font-montserrat` after the configuration of the default font family

**Review by danielerat — APPROVED** [review]
Great work for this tedious change Miss @CynthiaUmuhire!
Now things are less redundant!

**Review by Moussa-Kalam — APPROVED** [review]
Everything looks okay.

**Review by gitarecy — APPROVED** [review]
Font codes look better/cleaner

**Pull request merged**